### PR TITLE
added free courses

### PR DESCRIPTION
**Pull request by kartikay23**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -283,7 +284,6 @@
 * [More Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/moredataminingwithweka/)
 * [The Analytics Edge](https://www.edx.org/course/analytics-edge-mitx-15-071x-3)
```
Comment (LuigiImVector) [inline courses/free-courses-en.md:286]:
```suggestion


```
Just a little error

**Review by kartikay23 — COMMENTED** [review]
Please verify the changes and let me know.

**Comment by eshellman** [comment]
should list the platform `- (Jovian)`

**Review by kartikay23 — COMMENTED** [review]
Updated changes by added the name of platform.
Thank you

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**